files/templates/email/default.html

```diff
@@ -6,405 +6,109 @@
-		}
-		body {
-			background-color: #EDF2F7;
-			color: #121213;
-		}
-		p {
-			color: #121213;
-		}
-		p.sub {
-			color: #6B6E76;
-		}
-		.email-wrapper {
-			width: 100%;
-			margin: 0;
-			padding: 0;
-			-premailer-width: 100%;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			background-color: #EDF2F7;
-		}
-		.email-content {
-			width: 100%;
-			margin: 0;
-			padding: 0;
-			-premailer-width: 100%;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-		}
-		
-		.email-masthead {
-			display: none;
-		}
-		.email-masthead_logo {
-			width: 94px;
-		}
-		.email-masthead_name {
-			font-size: 1.25rem;
-			font-weight: bold;
-			color: #121213;
-			text-decoration: none;
-		}
-		
-		.email-body {
-			width: 100%;
-			margin: 0;
-			padding: 0;
-			-premailer-width: 100%;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			background-color: #cfcfcf;
-		}
-		.email-body_inner {
-			width: 570px;
-			margin: 0 auto;
-			padding: 0;
-			-premailer-width: 570px;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			background-color: #cfcfcf;
-		}
-		.email-footer {
-			width: 570px;
-			margin: 0 auto;
-			padding: 0;
-			-premailer-width: 570px;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			text-align: center;
-		}
-		.email-footer p {
-			color: #6B6E76;
-		}
-		.body-action {
-			width: 100%;
-			margin: 30px auto;
-			padding: 0;
-			-premailer-width: 100%;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			text-align: center;
-		}
-		.body-sub {
-			margin-top: 25px;
-			padding-top: 25px;
-			border-top: 1px solid #E6E6E6;
-		}
-		.content-cell {
-			padding: 35px;
-		}
-		
-		@media only screen and (max-width: 600px) {
-			.email-body_inner,
-			.email-footer {
-				width: 100% !important;
-			}
-		}
-		@media (prefers-color-scheme: dark) {
-			body,
-			.email-body,
-			.email-body_inner,
-			.email-content,
-			.email-wrapper,
-			.email-masthead,
-			.email-footer {
-				background-color: #121213 !important;
-				color: #FFF !important;
-			}
-			p,
-			ul,
-			ol,
-			blockquote,
-			h1,
-			h2,
-			h3 {
-				color: #FFF !important;
-			}
-			.attributes_content,
-			.discount {
-				background-color: #222 !important;
-			}
-			.email-masthead_name {
-				text-shadow: none !important;
-			}
-		}
-		</style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f5f5f5;
+        }
+        .container {
+            max-width: 800px;
+            margin: 0 auto;
+            padding: 20px;
+            background-color: #ffffff;
+            border-radius: 5px;
+            box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+        }
+        h1 {
+            text-align: center;
+            color: #333333;
+        }
+        p {
+            margin-bottom: 10px;
+            color: #333333;
+            line-height: 1.25rem;
+        }
+        .button-container {
```

Comment:
The centered button with `.button-container` and `.button` looks perfect in Outlook.com webmail, though it'd be nice if someone could test this on a Windows desktop client (like Outlook 2016 or Windows 10 Mail). Part of why the `<table>` junk lasted so long is it's the reliable way to get desktop Outlooks to align things; their HTML engine is famously deficient.

Action item here would be if someone with a Windows box could test this on an Outlook, though it may not be a blocker to merge even if it degrades desktop Outlook. Markup is simple enough it should be usable in any case. @zorbathut

Reply:
Well, I researched a bit but just went with it, wasn't sure how important compatibility with those are. I used mailtrap io page for the test which has a "market support" analysis and apparently the most problematic properties are box-shadow and rem for Outlook. Text-align should be supported for centering stuff (https://www.caniemail.com/features/css-text-align/)

Either case, I'll check if I can test with some of those clients or could just rewrite with tables

Reply:
The thing that looks potentially problematic to me is `.button { display: inline-block }`. If I remember correctly, Outlook will probably leave anchor tags as `display: inline`. This should still text align in theory, so perhaps it's alright; I just remember doing `<td align="center">` back in the day, and maybe compat is better now.

The code is certainly much nicer with normal HTML instead of the tables, it already displays better than the old templates, and the button failing to left-aligned is not a major problem. I'll mark approved and we can deal with it later if Outlook turns out to have actual usability problems.